lib/dbservice/enrollment_record/enrollment_record.ex

```diff
@@ -5,18 +5,17 @@ defmodule Dbservice.EnrollmentRecords.EnrollmentRecord do
   import Ecto.Changeset
   import Dbservice.Utils.Util
 
-  alias Dbservice.Users.Student
+  alias Dbservice.Users.User
```

Comment:
I think an `enrollment_record` should be of a `student` and not the `user`. What is the reason of associating it with user?